examples/image_warping_example.py

```diff
@@ -0,0 +1,83 @@
+"""
+Mouse event callbacks
+=====================
```

Comment:
```suggestion
Image warping with mouse event callbacks
========================================
```
I think that's a bit more descriptive for the title in the gallery.
Open to other ideas.
Not sure I counted the chars right for the number of `=`!